src/Juvix/Syntax/Abstract/Name.hs

```diff
@@ -16,6 +16,7 @@ data Name = Name
   { _nameText :: Text,
     _nameId :: NameId,
     _nameKind :: NameKind,
+    _namePretty :: Text, -- How to print this name in error messages
```

Comment:
is this really neccessary? I'd expect a function (lens) to reconstruct such a (pretty) name on the fly. Not to carry with that info everywhere.

Reply:
Can we remove it and make a function to get the same result?

Reply:
I don't see a way to do that. `_nameText` has to contain the unqualified name since it is used for compilation. Can you implement the function that you are referencing?

Reply:
Maybe. It was a question. ;)